cime_config/tests.py

```diff
@@ -121,6 +121,18 @@
             )
         },
 
+    "e3sm_cryo_developer" : {
+        "tests"   : (
+            "SMS_D_Ld1.TL319_IcoswISC30E3r5.GMPAS-JRA1p5-DIB-PISMF.mpaso-jra_1958",
```

Comment:
This test is probably too slow.  It took 15 minutes or so on 4 Chrysalis nodes.  We probably want to shorten it or not run this one in debug mode.  (I was roughly following the `e3sm_ice_developer` suite above.)

Reply:
I think this is ok. I ran the test suite with this and with `Ln5`, and both took about 40 minutes, so I think while this is running the suite is still doing other builds or whatnot.

Reply:
Sorry I was thinking in the context of an individual running this on one's own, not as part of the e3sm_developer suite. FWIW the 5 timestep test was 75 seconds runtime (+2 minutes init).

Reply:
@proteanplanet, do you have an opinion about running this test in debug mode or not and for 1 day vs. a shorter time?